File: 24-Python/Innovazione.py
from Progetto import Progetto

class Innovazione(Progetto):
    def __init__(self, codice, tipo, titolo, nomeCognome, coordinatore, milioni, numAziende) -> None:
        super().__init__(codice, tipo, titolo, nomeCognome, coordinatore, milioni)
        self._numAziende = numAziende

    def __str__(self) -> str:
        return super().__str__() +"\t-\t-\t"+ str(self._numAziende) + str(self._milioni * 1000)

File: 24-Java/src/Innovazione.java
public class Innovazione extends Progetto{
    private int numAziende;

    public Innovazione(int codice, String tipo, String titolo, String nomeCognome, String organizzazione,
            double milioni, int numAziende) {
        super(codice, tipo, titolo, nomeCognome, organizzazione, milioni);
        this.numAziende = numAziende;
    }

    public String toString(){
        return super.toString() +"\t-\t-\t"+ numAziende +"\t"+ super.milioni*1000;  
    }
}
